deploy/infra/aws/cloudformation/jump-server.yaml

```diff
@@ -0,0 +1,364 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Create Jump-box EC2 instance
+
+Parameters:
+  ProjectName:
+    Type: String
+    Description: "Name of the deployment.Will be used for resources name prefix"
+  Env:
+    Type: String
+    Description: "Environment name.Will be used for resources name prefix "
+  InstanceType:
+    Type: String
+    Description: Jump-server EC2 instance type
+    Default: "t3.medium"
+  AmiId:
+    Type: 'AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>'
+    Default: '/aws/service/eks/optimized-ami/1.29/amazon-linux-2/recommended/image_id'
+  VpcId:
+    Type: String
+    Description: "Id of the vpc where instance will be created"
+    Default: ""
+  SubnetId:
```

Comment:
Better to use more specific naming for this parameted, because it is not clear host this value will be used, f.i. `CPInfraSubnetId` would be better + provide good description for all parameters within this file